helpers/cubes.py

```diff
@@ -35,6 +35,34 @@ def load_input_cube(src, varname):
     return leg_cube
 
 
+DEFAULT_UNIT_CONVERSIONS = {
+    "kelvin": "degC",
+    "meter^-2-kilogram-second^-1": "meter^-2-kilogram-day^-1",
+}
+
+
+def convert_units(cube: iris.cube.Cube, conversions=None) -> iris.cube.Cube:
```

Comment:
Sorry, this fails for me:
````
    def convert_units(cube: iris.cube.Cube, conversions=None) -> iris.cube.Cube:
                            ^^^^^^^^^
AttributeError: module 'iris' has no attribute 'cube'
````

Reply:
Hej @klauswyser, do you have Python3.7 in your environment? Sounds like #101.

Reply:
Nope:
````
(scriptengine) [sm_wyser@freja1 run-monitor]$ python -V
Python 3.13.0
````
However, I do get a lot of iris-related warnings when running (without the convert_units commit) that I just ignore. Maybe iris needs an update? But I just installed a new env with all the packages, doesn't help.
````
# Name                    Version                   Build  Channel
scitools-iris             3.10.0                   pypi_0    pypi
````
Could it be that "my" iris is from pypi and not conda-forge?

Reply:
See PR #102.

Reply:
No, it was my fault and a real issue (not Python version related). PyPI or conda-forge should not make any difference. The only thing that matters should be the version (but not in this case).